dotcom-rendering/src/paletteDeclarations.ts

```diff
@@ -2976,7 +2989,7 @@ const articleBackgroundLight: PaletteFunction = ({
 		case ArticleDesign.FullPageInteractive:
 			return 'transparent';
 		case ArticleDesign.Gallery:
-			return sourcePalette.neutral[7];
+			return 'rgba(0,0,0,.25)';
```

Comment:
Should we use a colour from the Source palette and the `transparentColour` method here for consistency? ie.

```js
transparentColour(sourcePalette.neutral[0], 0.25)
```